### ISSUE TITLE
Add architectural considerations to the readme

### ISSUE THREAD

**Issue by denesnori** [CYPIAPT-LNDSE/sdm-app#52]
From Mark: "architectural considerations needed for the MVP (e.g. how is patient information needed, how to manage accounts)"

Can we discuss this? I'm uncertain what we should write about this.

**Comment by MHemsley** [comment]
Of course, I'm available on 07375043285 between 2 and 2:30, 3 and 4, and after 4:30.

**Comment by denesnori** [comment]
Hi Mark,
Thanks for the prompt reply. We have a full-day workshop today, so we would prefer if you can explain  us on github what you meant by "how to manage accounts?".

**Comment by MHemsley** [comment]
Of course, it was just a generic example thinking about previous projects, I wouldn't worry too much about the examples, as I did just pluck them out of the air

**Comment by denesnori** [comment]
Thanks Mark :) We will finish the readme tonight, hopefully.
